packages/frontend/core/src/components/affine/affine-error-eoundary.tsx

```diff
@@ -91,6 +94,18 @@ export class AffineErrorBoundary extends Component<
             </>
           </>
         );
+      } else if (error instanceof SessionFetchErrorRightAfterLoginOrSignUp) {
+        errorDetail = (
+          <>
+            <h1>Sorry.. there was an error</h1>
+            <>
+              <span> Fetching session failed </span>
+              <span onClick={error.refetchSession}>
```

Comment:
![image](https://github.com/toeverything/AFFiNE/assets/9527419/67fe5956-2074-4440-899d-d31f93b8a904)

This click is too hard to use.

Reply:
Use `<a>` to replace `<span>`, and we need to implement the design after this PR. https://www.figma.com/file/UTTtfVCcStw1dKUg0gCSPo/%F0%9F%93%9A-Design-System?type=design&node-id=11684-4096&mode=design&t=EnURvqBaeKkWsQAo-0

Reply:
I am thinking. We should suggest user to refresh the page after too much retrying.